pkg/controller/containerizedworkload/manualscaler.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package containerizedworkload
```

Comment:
why is a trait under a workload dir?

Reply:
It is under `containerizedworkload` because the functionality here is specific to how we want the `ManualScaler` to modify a `ContainerizedWorkload` that has been packaged into a `KubernetesApplication`. It is not clear that a `ManualScaler` will work in this same manner on every workload type that it applies to. However, I would think that `ManualScaler` (compared to other trait kinds) could be a candidate for modifying every workload type it is compatible with in the same way (i.e. by changing the `ReplicaCount` field on a `Deployment`).